msteams-platform/bots/overview.md

```diff
@@ -0,0 +1,99 @@
+---
+title: What is a bot
+description: Learn about bots, the application of bots across industry, build familiarity with bots in Teams environment and bot development SDKs and libraries. It offers a map through the module to help you navigate through various tasks for building a bot.
+ms.localizationpriority: high
+ms.date: 09/20/2024
+ms.topic: conceptual
+ms.author: surbhigupta
+---
+
+# Bot overview
+
+A bot is a software application that performs automated tasks by interacting with systems or users. From simple automation tool to sophisticated artificial intelligency (AI) powered apps, bots have transformed the way users interact with technology, streamlining tasks, and enhancing efficiency.
+
+You can leverage bot capabilities such as chatbots, virtual assistants, workflow, and have diverse applications across industries such as, healthcare, customer service, education,finance, and more. Bots bring efficiency in automation, scalability, continual availability, and with AI bots you can offer tailored experiences for the users.
+
+## How does building a bot benefit me?
+
+Bots can save you time and effort by handling tasks that would otherwise take up your time. They can work 24/7 without getting tired, and they can handle multiple tasks at once. Bots transform how developers engage with users, streamline tasks, and boost efficiency. As a developer, you'll craft interactive and intelligent solutions to enhance user experiences.
+
+The bot app users access and use your bot app within Teams.
+
+[**TBD**: Bot UX infographic or gif]
+
+Building a bot can offer benefits in the following scenarios:
```

Comment:
This list speaks more to benefits rather than specific user scenarios. How about something like:

Bots apps can provide the following business benefits:

Reply:
@erikadoyle I've reworded as suggested.